doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,343 @@
+.. _pymapdl_docker:
+
+****************
+MAPDL and Docker
+****************
+
+You can run MAPDL within a container on any OS using Docker and
+connect to it via PyMAPDL.
+
+There are several situations in which it is advantageous to run MAPDL
+in a containerized environment such as Docker or Singularity:
+
+- Run in a consistent environment regardless of the host OS.
+- Portability and ease of install.
+- Large-scale cluster deployment using Kubernetes
+- Genuine app isolation through containerization
```

Comment:
```suggestion
There are several advantages to running MAPDL
in a containerized environment such as Docker or Singularity:

- Consistent environment regardless of the host OS
- Portability and ease of install
- Large-scale cluster deployment using Kubernetes
- Genuine app isolation through containerization
```
Title of section is PyMAPDL and Docker. Is there a reason to mention Singularity here? It seems out of place.

Reply:
Hi @PipKat! Yes, it's a container technology so we should keep "Singularity"